eng/testing/.runsettings

```diff
@@ -18,10 +18,6 @@
     <!-- Filter out failing (wrong framwork, platform, runtime or activeissue) tests -->
     <TestCaseFilter>$$TESTCASEFILTER$$</TestCaseFilter>
     <DotNetHostPath>$$DOTNETHOSTPATH$$</DotNetHostPath>
-    <EnvironmentVariables>
-      <!-- Use our self-built framework on .NET Framework -->
-      <DEVPATH>$$DEVPATH$$</DEVPATH>
```

Comment:
Why is this not needed anymore? We do have some implementations that we built and that are not on the GAC.

Reply:
This isn't needed anymore as all .NETFramework libraries are app-local now. There is now need for the additional runtime directory anymore. Assembly binding redirects handle the cases where we target a live build asset that is present in the framework and will prefer ours same as in a customer app. This means that we now test the same way our customers do which is a nice plus.